### PR TITLE
DocumentDB List Connection Strings Action

### PR DESCRIPTION
**Pull request by dmakwana**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [X] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

### CODE REVIEW

File: arm-documentdb/2015-04-08/swagger/documentdb.json [hunk 1]
```diff
@@ -280,6 +280,34 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/listConnectionStrings": {
+      "post": {
```
Comment (vishrutshah) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: Curious: Why is this post action? Should it be get action?

**Review by vishrutshah — CHANGES_REQUESTED** [review]
@dmakwana Is this the new endpoint just been released or the swagger was missing this endpoint with this api-version?

File: arm-documentdb/2015-04-08/swagger/documentdb.json (hunk 1 above)
Comment (dmakwana) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: All database account actions are set up as POST actions in DocumentDB

**Comment by dmakwana** [comment]
This endpoint is new, in the same api-version. It was missing in the current swagger document

File: arm-documentdb/2015-04-08/swagger/documentdb.json (hunk 1 above)
Comment (amarzavery) [inline arm-documentdb/2015-04-08/swagger/documentdb.json]: @vishrutshah - This is the same as StorageAccounts_ListKeys (which is a POST operation).

**Comment by vishrutshah** [comment]
@dmakwana Changes looks good. It'd be great if you can please provide x-ms-examples for atleast this operation. That'd help us in verifying the operation and it's model via "Swagger Model Validator" tool that we are building. 

https://github.com/Azure/adx-documentation-pr/blob/master/tools/tools.md#swagger-model-validation

**Comment by amarzavery** [comment]
Hi @dmakwana, 

Is it possible for you to provide x-ms-examples for the doc db swagger? This will help us validate the models and the examples will be a part of the docs.microsoft.com (REST API doc) website. It will be very useful for our customers. It would be great if you could provide a minimum and a maximum example for every operation in the spec.

I understand that it will be a little extra work right now. However, moving forward the delta would be relatively as big as the new changes being made.

Please follow [this link](https://github.com/Azure/adx-documentation-pr/blob/master/tools/tools.md#swagger-model-validation) and let me know if you have any questions.

**Comment by dmakwana** [comment]
I've added an example for the listConnectionStrings action. Question: the response is different for differently configured database accounts, how do we make a request for a differently configured database account? Also, if the response has any unpredictable values (hashed values), how can we mask those?

**Comment by dmakwana** [comment]
I can add the rest of the examples in a separate PR if you don't mind

**Comment by amarzavery** [comment]
@vishrutshah - FYI, synced up with @dmakwana  on skype. He will be providing example for connectionStrings in this PR. Examples for other operations would come in another PR.

File: arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountListConnectionStrings.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "parameters": {
+    "name": "ddb1",
```
Comment (vishrutshah) [inline arm-documentdb/2015-04-08/examples/DocumentDBDatabaseAccountListConnectionStrings.json]: Should this be `accountName` instead of `name`

**Review by vishrutshah — COMMENTED** [review]
@amarzavery Waiting for you data model validation to pass.

**Review by amarzavery — APPROVED** [review]

**Review by vishrutshah — APPROVED** [review]

**Pull request merged**